modules/gateway.go

```diff
@@ -40,11 +25,33 @@ type NetConn interface {
 // RPCFunc is the type signature of functions that handle incoming RPCs.
 type RPCFunc func(NetConn) error
 
+// A NetAddress contains the information needed to contact a peer.
+type NetAddress string
+
 type GatewayInfo struct {
 	Address NetAddress
 	Peers   []NetAddress
 }
 
+// TODO: Move this and it's functionality into the gateway package.
+var (
+	BootstrapPeers = []NetAddress{
+		"23.239.14.98:9988",
+	}
+)
```

Comment:
true...but we could :P
Alternatively, we could move the hard-coded addresses out of the code entirely, and load them from the config file instead.